src/styles/blocks/normalize.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul,
+li,
+p,
+a,
+h2,
+h1 {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+}
```

Comment:
`a` has no `list-style` property, and `h1` or `h2` so check each element and reset exactly the styles that should be reset for it